websites/J/Juniperbot/presence.ts

```diff
@@ -0,0 +1,64 @@
+const 
+  presence = new Presence({clientId: "739908991274057870"}),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {largeImageKey: "logo"},
+    juniper: boolean = await presence.getSetting("juniper"),
+    docs: boolean = await presence.getSetting("docs"),
+    fback: boolean = await presence.getSetting("fback");
```

Comment:
add defaults:
variable: boolean = await presence.getSetting("var") || true

This is needed due to a bug we currently have.